README.md

```diff
@@ -133,6 +138,9 @@ Older browsers like IE11 run LWC in compatibility mode. For more information abo
 
 ### Deprecated
 
-| Rule ID                                                            | Replaced by                                                                  |
-| ------------------------------------------------------------------ | ---------------------------------------------------------------------------- |
-| [lwc/no-dupe-class-members](./docs/rules/no-dupe-class-members.md) | [no-dupe-class-members](https://eslint.org/docs/rules/no-dupe-class-members) |
+| Rule ID                                                                                                  | Replaced by                                                                                |
+| -------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------ |
+| [lwc/no-dupe-class-members](./docs/rules/no-dupe-class-members.md)                                       | [no-dupe-class-members](https://eslint.org/docs/rules/no-dupe-class-members)               |
```

Comment:
```suggestion
| [lwc/no-dupe-class-members](./docs/rules/no-dupe-class-members.md)                                       | [no-dupe-class-members](https://eslint.org/docs/rules/no-dupe-class-members) (base eslint rule)               |
```
This one's different from the others because it's not a rule from this plugin. We should call that out.